olp-cpp-sdk-dataservice-write/test/unit/TestStreamLayerClient.cpp

```diff
@@ -1610,7 +1620,6 @@ TEST_P(StreamLayerClientCacheOnlineTest,
 
   EXPECT_LE(3, default_listener->GetNumFlushEvents());
   EXPECT_LE(3, default_listener->GetNumFlushEventsAttempted());
-  EXPECT_EQ(0, default_listener->GetNumFlushEventsFailed());
```

Comment:
Better to not remove this line, but leave it commented with description why this check was removed.